Alters "Get Started" section (#1065)
iotaledger/iota-wiki: docs/get-started/introduction/shimmer/introduction.md

---
description: "Discover Shimmer, IOTA's staging network designed to test and deploy innovative features like tokenization and smart contracts. Explore Shimmer's capabilities and understand its role as a testing ground for advancements before their implementation on the IOTA network."
image: /img/docssection/shimmer_dark.svg
keywords:
  [
    Shimmer,
    IOTA,
    staging network,
    tokenization,
    smart contracts,
    canary network,
    innovation,
    Shimmer Devnet,
  ]
---

# Introduction

> Shimmer acts as the production grade staging network for the IOTA mainnet where new protocol changes are introduced and

Reviewer: ```suggestion
Shimmer acts as the production-grade staging network for the IOTA mainnet where new protocol changes are introduced and
```

evaluated for functionality, scalability, and security before they are made available on the IOTA mainnet. However,

> unlike most staging networks, Shimmer has a [token called SMR](/get-started/introduction/shimmer/shimmer-token/) which

Reviewer: ```suggestion
unlike most staging networks, Shimmer has a [token called SMR](/get-started/introduction/shimmer/shimmer-token/), which
```

is traded on major exchanges, thus giving the network real monetary value, which in turn is beneficial to foster the
testing of upcoming protocol changes even further.

Currently, Shimmer runs the [Stardust protocol](/learn/stardust/introduction/) version, enabling tokenization and the anchoring of L2 smart contract
chains into L1.

> Shimmer is also the network on which the ShimmerEVM runs on, an L2 solution which provides smart contract

Reviewer: ```suggestion
Shimmer is also the network on which the Shimmer EVM runs, an L2 solution that provides smart contract
```
